.github/workflows/Tests.yml

```
@@ -1410,8 +1358,14 @@ jobs:
         python-version: "${{ vars.PYTHON_DEFAULT }}"
     - name: Prep Testing Tox
       id: prep-tox
-      run: |
-        if [ "$OS" == "ubuntu-latest" ] ; then { sudo apt-get update || true ;} ; wait ; { sudo apt-get install --assume-yes python3.10 python3.11 python3.11-venv || echo "::warning file=.github/workflows/Tests.yml,line=482,endLine=482,title=SKIPPED::SKIP Enhanced TOX Tests." ;} ; wait ; fi
+      uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)